tools/run-ui-test-with-mock-lightspeed-server.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# For invoking VS Code executable to open "vscode:" URL
+export UI_TEST=1
```

Comment:
Are you confident `1` equates to [`true`](https://github.com/ansible/vscode-ansible/pull/1405/files#diff-ff27543dbc8fdf6d075f959784b5efa93483e056a7dd470da27ba87d23b45bf6R8) in TypeScript?

I know this is a common Python pattern... but since TypeScript is type-safe using a `boolean` may be better?

Reply:
This is a bash shell script and I think using `1` as `true` is a convention in shell programming.

Besides the convention, using an environment variable here itself may not be a good programming practice... Probably I should implement a command option for mock lightspeed server. I will consider that when I implement MacOS support. Thanks.